tests/WP_Ultimo/SSO/Admin_Bar_Magic_Links_Test.php

```
@@ -60,4 +60,264 @@ public function test_modify_my_sites_menu_no_user(): void {
+		$this->assertNotWPError($mapping);
+
+		$_REQUEST = array( // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Sets the handler request fixture.
+			Admin_Bar_Magic_Links::SITE_ID_QUERY_ARG => (string) $site_id,
+			'_wpnonce'                               => wp_create_nonce(Admin_Bar_Magic_Links::ADMIN_POST_ACTION . '_' . $site_id),
+			'redirect_to'                            => 'https://attacker.example.test/',
+		);
+
+		$magic_link        = 'https://admin-bar-magic-links.example.test/wp-admin/?wu_magic_token=test-token';
+		$redirect          = array();
+		$redirect_to       = '';
+		$magic_link_filter = static function ($url, $filter_user_id, $filter_site_id, $filter_redirect_to) use (&$redirect_to, $magic_link) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$url'. (undefined)

(UnusedFormalParameter)